docs/ossm/ossm2-migration/README.md

```diff
@@ -0,0 +1,20 @@
+# OSSM 2.x migration to 3.x
+
+This document details how to migrate from OSSM 2.x to OSSM 3.x.
+
+## Pre-migration Checklist
+
+Before you begin to migrate your controlplane from OSSM 2.x to 3.x, ensure you have done the following:
+
+- Updated your `SMCP` version to the latest OSSM release.
+- You have disabled the following features on your SMCP. These fields are unsupported in 3.0 and must be disabled prior to migration.
+  - Network Policy management: `spec.security.manageNetworkPolicy=false`
+  - Disabled addons:
+    - Prometheus: `spec.addons.prometheus.enabled=false`
+    - Kiali: `spec.addons.kiali.enabled=false`
+    - Grafana: `spec.addons.grafana.enabled=false`
+    - Tracing: `spec.tracing.type=None`
+
+## MultiTenant
+
+If you are migrating a `MultiTenant` controlplane to OSSM 3.0, follow [these instructions](./multi-tenancy/README.md).
```

Comment:
Is `MultiTenant` a configuration value somewhere? If you asked most OSSM users, who are running a single mesh per cluster after following all of our defaults, they would tell you they aren't using multi-tenancy... so, we should add a step on how to validate if you are using "multi-tenant" or "cluster wide" mode and then direct people as appropriate.  Also, the person who setup the mesh may not be the person upgrading it.

Reply:
Yes it's a setting on the `SMCP`: `spec.mode: MultiTenant`. I'll add a step for users to verify this. I made add this to the "pre-flight" check list.